shark_turbine/kernel/wave/distribution_symbols.py

```diff
@@ -0,0 +1,9 @@
+import shark_turbine.kernel.lang as tkl
+
+WORKGROUP_0 = tkl.sym.WG0
+WORKGROUP_1 = tkl.sym.WG1
+WORKGROUP_2 = tkl.sym.WG2
+
+THREAD_0 = tkl.sym.T0
```

Comment:
General comment - I think, defining internal symbols in that manner may not be safe as user can accidentally define symbols with the same name in his code and break things in unexpected ways (especially for short and common names).
We probably need separate facility to create internal symbols while will never overlap with `tkl.sym` (e.g. by prepending sympy symbol name with "$").

Reply:
Good point. I will update this in the next PR.